prov/efa/test/efa_unit_test_ep.c

```diff
@@ -127,7 +127,7 @@ void test_efa_rdm_ep_handshake_exchange_host_id(struct efa_resource **state, uin
 	assert_non_null(peer);
 	/* Peer host id is uninitialized before handshake */
 	assert_int_equal(peer->host_id, 0);
-	assert_false(peer->flags && EFA_RDM_PEER_HANDSHAKE_SENT);
+	assert_false(peer->flags & EFA_RDM_PEER_HANDSHAKE_SENT);
```

Comment:
how about adding a `!!` here? it's now a uint64 I guess.

Reply:
Yeah so there's definitely some weirdness with pedantically asserting `false` for a masked `uint32_t`. It's probably ok in this scenario since the expression can only evaluate to `0` or `1` since `EFA_RDM_PEER_HANDSHAKE_SENT` is `1`. I'll just change this to an `assert_int_not_equal()` call.

Side note: CMocka's suite of `assert_*` macros is actually lacking a little. There are macros for comparing `int` and `double`, but that's really it for primitive values. Unity has [corresponding macros](https://github.com/ThrowTheSwitch/Unity/blob/master/docs/UnityAssertionsReference.md#signed-and-unsigned-integers-of-all-sizes) for just about everything in `stdint.h`